Src/Campus.Master.API/UI/src/assets/locale/ua.json

```diff
@@ -64,6 +64,11 @@
             "SIGN-IN": "Вхід",
             "NOT-REGISTERED": "Ще не зареєстровані?",
             "SIGN-UP": "Зареєструватися"
+        },
+        "ERROR-TOASTR": {
+            "HEADER": "Помилка",
```

Comment:
Add Ukrainian localization for custom messages

Reply:
Done